app/models/concerns/taxonomix.rb

```diff
@@ -173,6 +173,8 @@ def children_of_selected_organization_ids
   # for existing records it would block saving global objects if taxonomies were not changed
   # for new records this would allow to create global objects for non global users
   def ensure_taxonomies_not_escalated
+    return true if self.respond_to?(:skip_taxonomy_escalation_check?) && self.skip_taxonomy_escalation_check?
```

Comment:
I think it's just wrong use of Taxonomix, Filter should not have included it in the first place, that's why I didn't want to put any exceptional logic to Filter (to make the problem worse) but it's not a big deal, I can wrap a method in Filter and verify the condition there if you prefer.